src/main/java/com/ays/user/model/mapper/UserToUserResponseMapper.java

```diff
@@ -24,4 +25,6 @@ public interface UserToUserResponseMapper extends BaseMapper<User, UserResponse>
     static UserToUserResponseMapper initialize() {
         return Mappers.getMapper(UserToUserResponseMapper.class);
     }
+    @Mapping(source = "supportStatus",target = "supportStatus")
+    UserResponse map(User user);
```

Comment:
Burada neden özel bir tanım yapmamız gerekti?

Reply:
Normal mappinde SuppotStatus null geliyordu,Herhangi bir mapping işlemi yapılmadığı için mecburen oraya özel tanım yapılması gerekildi.